fbchat/utils.py

```diff
@@ -26,6 +26,7 @@
     check_request,
     get_jsmods_require,
     get_emojisize_from_tags,
+    get_forwarded_from_tags,
```

Comment:
Hmm, I actually made this file so that it would be easy to see when we made backwards incompatible changes, not just to re-export everything from `_util.py`. And I don't think I want to make this function public, since it's mostly an implementation detail.